config/samples/falcon_v1alpha1_falconcontainer-all-options.yaml

```diff
@@ -0,0 +1,29 @@
+# Falcon Container Installation.
+#
+# Example using all/many available options
+---
+apiVersion: falcon.crowdstrike.com/v1alpha1
+kind: FalconContainer
+metadata:
+  name: default
+spec:
+  falcon_api:
+    client_id: PLEASE_FILL_IN
+    client_secret: PLEASE_FILL_IN
+    cloud_region: autodiscover
+    # (optional, default is derived from the API Key pair)
+    cid: 00001111222233334444555566667777-12
+  injector:
+    sa_annotations:
+      eks.amazonaws.com/role-arn: arn:aws:iam::111122223333:role/iam-role-name
```

Comment:
Do you want to include the GCP counter parts as well? It is documented at https://github.com/CrowdStrike/falcon-operator/blob/main/docs/deployment/gke/gcp-workload-identity.md